kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/listeners/LoggingStateRestoreListener.java

```diff
@@ -38,7 +38,7 @@ public void onBatchRestored(
     long currentProgress = batchCompleted + restoredSoFar.getOrDefault(topicPartition, 0L);
     double percentComplete = (double) currentProgress / totalToRestore.get(topicPartition);
 
-    LOGGER.info(
+    LOGGER.debug(
```

Comment:
these logs are helpful in tracking start/end/progress of restoration of state from changelog topics.

Reply:
We can turn them on if we need to debug right? Right now they are logging too much.